tsconfig.json

```diff
@@ -12,7 +12,8 @@
         "experimentalDecorators": true,
         // "strict": true,
         "strictNullChecks": true,
-        "strictFunctionTypes": false
+        "strictFunctionTypes": false,
+        "allowJs": true
```

Comment:
Thanks for your PR!
Could you explain why this change is useful/neccesary?

Reply:
As I write in the commit message of 167f151800dbb8c5429fc63dd7342a9a1e91ec55:

> Because `hardhat.config.js` is listed in `tsconfig.json`'s `files`,
`allowJs` must be enabled

Reply:
Have updated hardhat config so this should no longer be necessary.

Reply:
I've removed the commit